<launch>
  <arg name="yamlFile" default="$(find abb_driver)/config/gofa_cfg.yaml" />
  <rosparam command="load" file="$(arg yamlFile)" />

  <group ns="Gofa">

    <!-- Load the URDF pkg -->
    <include file="$(find robots_description)/launch/display.launch">
      <arg name="robot" value="gofa" />
    </include>

   
    <node pkg="abb_driver" name="abb_interface" type="abb_interface_node" output="screen" />
    <node pkg="abb_driver" name="controller_bridge" type="controller_bridge_node" output="screen" />
    <node pkg="abb_controllers" name="OneTaskInvKin" type="OneTaskInvKin_node" output="screen" />
  </group>

</launch>